cpu/stm32l4/vectors.c

```diff
@@ -172,32 +170,41 @@ ISR_VECTOR(1) const isr_t vector_cpu[CPU_IRQ_NUMOF] = {
     [TIM1_TRG_COM_IRQn       ] = isr_tim1_trg_com,         /* [26] TIM1 Trigger and Commutation Interrupt */
     [USB_IRQn                ] = isr_usb,                  /* [67] USB event Interrupt */
     [CRS_IRQn                ] = isr_crs,                  /* [82] CRS global interrupt */
-#elif defined(CPU_MODEL_STM32L476RG) || defined(CPU_MODEL_STM32L475VG)
+#endif
+#if defined(CPU_MODEL_STM32L476RG) || defined(CPU_MODEL_STM32L475VG) || \
+      defined(CPU_MODEL_STM32L452RE)
     [ADC1_2_IRQn             ] = isr_adc1_2,               /* [18] ADC1, ADC2 SAR global Interrupts */
```

Comment:
This enum doesn't exist in l452 cmsis file, it's called ADC1_IRQn, like for the l432.

Reply:
there an alias defined in the cmsis files at line 16034, would you rather have me use the original one?

```
/* Aliases for __IRQn */
#define ADC1_2_IRQn                    ADC1_IRQn
```

Reply:
> there an alias defined in the cmsis files at line 16034

Sorry, I didn't scroll that far ;)

> would you rather have me use the original one?

I would say yes, we never know. It may simplify an update of that file if one day there's a new version from ST.

Reply:
I think I got confused jajajaja, should or should I not use the alias. (I wasn't very clear what I meant by original one jajajaja)

Reply:
Do you mean the alias was already there (ST code) or you added it for convenience ? If it's the latter, I would avoid this, otherwise, keep it like this.

Reply:
It was included in the original cmsis file, with this header 

>/******************************************************************************/
/*  For a painless codes migration between the STM32L4xx device product       */
/*  lines, the aliases defined below are put in place to overcome the         */
/*  differences in the interrupt handlers and IRQn definitions.               */
/*  No need to update developed interrupt code when moving across             */
/*  product lines within the same STM32L4 Family                              */
/******************************************************************************/
So I think it's going to stay in future files too.